src/main/java/me/zeroeightsix/kami/module/modules/movement/EntitySpeed.kt

```diff
@@ -60,8 +61,8 @@ class EntitySpeed : Module() {
         val right = mc.gameSettings.keyBindRight.isKeyDown
         val back = mc.gameSettings.keyBindBack.isKeyDown
 
-        if (!(forward && back)) boat.motionY = 0.0
-        if (mc.gameSettings.keyBindJump.isKeyDown) boat.motionY += speed.value / 2f.toDouble()
+        if (!(forward && back)&&flight.value) boat.motionY = 0.0
+        if (mc.gameSettings.keyBindJump.isKeyDown&&flight.value) boat.motionY += speed.value / 2f.toDouble()
```

Comment:
format this properly please

Reply:
Not sure how it should be formated but I think I know what you mean.

Reply:
spaces around `&7`